code/game/objects/items/reagent_containers/autoinjectors.dm

```diff
@@ -218,14 +232,14 @@
 
 /obj/item/reagent_containers/hypospray/autoinjector/isotonic
 	name = "isotonic solution autoinjector"
-	desc = "An auto-injector loaded with 2 doses of isotonic solution, formulated to quickly recover fluid volume after blood loss or trauma."
```

Comment:
why?

Reply:
nic solution injector lowered to 25u, single-dose. Injectors are supposed to cap at 30u max, 50 is powercreep. One injector should recover roughly 15-20% of your blood. Injector added to Ma

Reply:
v

Reply:
I changed it cause I wanted to.

You want me to change it back, gimme a reason to, and we can discuss that.